lib/streamlit/elements/widgets/file_uploader.py

```diff
@@ -225,7 +229,7 @@ def file_uploader(
         *,  # keyword-only arguments:
         disabled: bool = False,
         label_visibility: LabelVisibility = "visible",
-    ) -> SomeUploadedFiles:
+    ) -> Optional[Union[UploadedFile, List[UploadedFile]]]:
```

Comment:
nit: same comment about adding `None` to the union instead of having this be `Optional` (same thing again with L389)

Reply:
done